packages/eas-cli/src/update/queries.ts

```diff
@@ -93,3 +189,33 @@ function renderUpdateGroups(
     Log.log(updateGroupsTable.toString());
   }
 }
+
+function renderUpdateGroupsAsTableWithBranchColumn(
+  updateGroups: (BranchUpdateGroupObject | AppUpdateGroupObject)[],
+  { json }: PaginatedQueryOptions
+): void {
+  const branch = {
+    name: updateGroups[0]?.[0]?.branch.name ?? 'N/A',
+    id: updateGroups[0]?.[0]?.branch.id ?? 'N/A',
+  };
+
+  const updateGroupDescriptions = getUpdateGroupDescriptionsWithBranch(updateGroups);
+
+  if (json) {
+    printJsonOnlyOutput({ ...branch, currentPage: updateGroupDescriptions });
+  } else {
+    const updateGroupsTable = new CliTable({
+      head: UPDATE_COLUMNS_WITH_BRANCH,
+      wordWrap: true,
+    });
+
+    updateGroupDescriptions.forEach(({ branch, message, runtimeVersion, group, platforms }) => {
```

Comment:
this should get the branch from each group?

Reply:
i think this is resolved?

Reply:
Ah yep, it's just a bit jarring that the inner branch destructured variable shadows the branch variable in the outer scope.